mbed_devices/_internal/resolve_target.py

```diff
@@ -1,19 +1,32 @@
 """Resolve targets for CandidateDevice.
 
-Filesystem access is slow - this module does its best to do it lazily.
+Resolving a target involves looking up an MbedTarget from the mbed-targets API, using data found in the "htm file"
+located on an "Mbed Enabled" device's USB MSD.
+
+Public Functions:
+    resolve_target: Resolves target data for the device.
```

Comment:
Correct me if I'm wrong, but I don't think this'll render correctly. I think lists need an empty line above them to render items correctly. Also worth wrapping names of public members in back ticks as it should link to their definitions in generated docs.

Reply:
I think you're probably right. I'm not sure if we even need the functions listed, after you informed me pdoc automatically does this on Slack.

Reply:
I don't think we generate docs for the internal modules, so I guess rendering is moot?

Reply:
That is true. 😅

Reply:
Still, we should probably have it so that it'd render correctly? _Consistency._ If we are to leave the docstring that is.

Reply:
Now that's just trolling.

Reply:
I'll consistetize it anyway (consistetize? another one for the glossary)

Reply:
There doesn't seem to be a way to generate output for private members, so it's not easily testable. I wouldn't bother tbh. You're right that it's trolling.

Reply:
So I don't think it needs a line space, but for an unordered list I do need "*" in front of each element.

Reply:
So, should I just leave it as it is?

Reply:
Removed the list of functions and exceptions in 552aa54